docs/node.md

```diff
@@ -61,8 +61,8 @@ Describes the resources available on the node: CPUs, memory and the maximum
 number of pods that can be scheduled on this node.
 
 ### Node Info
-General information about the node, for instance kernel version, kubernetes
-version, docker version (if used), OS name. The information is gathered by 
+General information about the node, for instance kernel version, kubelete version,
+kube-Proxy version, docker version (if used), OS name. The information is gathered by 
 Kubernetes from the node.
```

Comment:
it should rather be 
```kubelet version, kube-proxy version```

However, I suggest:
```kubernetes version (kubelet version, kube-proxy version)```

Or just leave it as it was before.